SURFnet.Authentication.sln

```diff
@@ -27,6 +27,17 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SURFnet.Authentication.Adfs
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SURFnet.Authentication.Adfs.Plugin.Test", "test\SURFnet.Authentication.Adfs.Plugin.Test\SURFnet.Authentication.Adfs.Plugin.Test.csproj", "{B0308356-431E-4C0C-8E15-DEE2BB16C0B0}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "DevTooling", "DevTooling", "{97EBA88D-8A52-4FCB-BB55-541B88BC07D2}"
+EndProject
+<<<<<<< HEAD
```

Comment:
Volgens mij staat hier nog een Head etc in ivm merge conflict

Reply:
Dit ziet er inderdaad uit als een mislukte merge

Reply:
Opgelost in nieuwe commit